.gitignore

```diff
@@ -124,3 +124,4 @@ Desktop.ini
 
 # IPython
 .ipynb_checkpoints/
+
```

Comment:
```suggestion
```

Reply:
You should be able to just commit this suggestion (I don't have write access to your branch)